_includes/postbox.html

```diff
@@ -8,11 +8,10 @@
                 {% if post.image %}
 
                     {% if site.lazyimages == "enabled" %}
-                        <img class="img-fluid lazyimg" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image contains "://" %}{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image }}{% endif %}" alt="{{ post.title }}">
+                        <img class="img-fluid lazyimg" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image-location contains "://" %}{{ site.image-location }}thumbnail_{{ post.image }}{% else %}{{ site.baseurl }}/{{ site.image-location }}thumbnail_{{ post.image }} }}{% endif %}" alt="{{ post.title }}">
```

Comment:
Is `post.image-location` correct here? It is different in other _includes files.